app/screens/settings-screen/security-screen.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from "react"
+import { useState } from "react"
+import { useFocusEffect } from '@react-navigation/native'
+import { Text, View } from "react-native"
+import { Button, Switch } from "react-native-elements"
+import EStyleSheet from 'react-native-extended-stylesheet'
+import RNSecureKeyStore, {ACCESSIBLE} from "react-native-secure-key-store"
+
+// Components
```

Comment:
Are these comments necessary?

Reply:
You're referring to the comments for the imports? I'm not sure what style we will end up going with for importing files. I'm agnostic on it. I was trying to copy the import styling I saw cryptosalomao using in some of his PRs.

Reply:
Yes that is what I'm referring too.

I would advocate to use as few comments as possible and instead try to make the code as self-explanitory as possible.
In these cases there is often a 1 word comment where the 1 word is contained in the path of what is being imported.
Like `themes` I don't need to read it in a comment. I can see it in the import path.

Communicating via code layout / structure (ie. dir and file names) is the way to go here! cc @cryptosalomao